src/ds/ds-widget.js

```diff
@@ -85,22 +94,31 @@ jQuery(function ($) {
                     casesensitive: false,
                     maxResults: 10,
                     itemEl: obj.options.entity_selector,
-                    emptyNode: obj.options.no_results,
                     getValue: function(that) {
                         let v = that.val();
                         let i = v.indexOf('@');
                         return i > -1 ? v.substring(i+1,v.length) : v;
                     },
-                    maxResultsNode: function(data) {
-                        return obj.options.too_many_results(data.length);
+                    sourceNodes: function(opts, val, results, render) {
+                        console.log(opts)
+                        console.log(val)
+                        console.log(results)
+                        console.log(render)
```

Comment:
Not sure if you wanted to keep the console logs here.

Reply:
no I will drop them when I merge the PR